integration-tests/java/README.md

```diff
@@ -28,4 +28,42 @@ You can run tests from the top level directory with the following command.
 
 ```bash
 $ ./gradlew integration-tests:java:integrationTest
-``` 
+```
+
+# How to initiate a new project from this example
+This section explains how you can write your Java clients based on this example.
+
+### Set up a new directory
+Create a new directory, and copy entire code Java integration tests to there.
+
+```bash
+$ mkdir my-client
+$ cd my-client
+$ copy -R PATH_TO_STELLARSTATION_API/integration-tests/java/* ./
+```
+
+### Replace a dependency in build.gradle
+In order to run those test outside of stellarstation-api, you need to replace the dependency
+to stellarstation-api from internal reference to external one.
+
+In order to do that, open `build.gradle` and replace `implementation project(':api')` in dependencies section to
```

Comment:
To do that, open `build.gradle` and replace `implementation project(':api')` in the dependencies section with `implementation 'com.stellarstation.api:stellarstation-api:0.4.0'`.